usr/share/applications/regolith.desktop

```diff
@@ -2,7 +2,7 @@
 Type=Application
 Name=regolith
 Comment=Productivity-focused desktop environment, Regolith
-Exec=regolith-session-init
+Exec=regolith-session-init i3
```

Comment:
interesting strategy.  I would there be a problem if we simply execute separate init scrips for each backend rather than having the script handle both?

Reply:
Actually no. Having separate scripts would work just the same.